dogpile/cache/region.py

```diff
@@ -32,6 +33,8 @@
 
 """
 
+log = logging.getLogger('c2cwsgiutils.' + __name__)
```

Comment:
Any reason to use `c2cwsgiutils` there?
The module name is not enough?

Reply:
Oups, missing to remove ...

Reply:
hehe no problem :)